src/HealthKit/Enums.cs

```diff
@@ -232,20 +254,39 @@ public enum HKCategoryTypeIdentifier {
 		[iOS (9,0)]
 		IntermenstrualBleeding,
 		[iOS (9,0)]
-		SexualActivity
+		SexualActivity,
+		[iOS (10,0), Watch (3,0)]
+		MindfulSession,
 	}
 
 	// Convenience enum, ObjC uses NSString
+	[Watch (2,0)]
 	[iOS (8,0)]
 	public enum HKCharacteristicTypeIdentifier {
 		BiologicalSex,
 		BloodType,
 		DateOfBirth,
 		[iOS (9,0)]
 		FitzpatrickSkinType,
+		[iOS (10,0), Watch (3,0)]
+		WheelchairUse,
+	}
+
+	// Convenience enum, ObjC uses NSString
+	[Watch (2,0), iOS (8,0)]
+	public enum HKCorrelationTypeIdentifier {
```

Comment:
it's a new `enum` so please use `[Field]` so the generator will handle the value-to-NSString conversions